README.md

```
@@ -1,23 +1,38 @@
 - Go to repository settings
 - Select Secrets and variables → Actions
 - Add new secrets with name
-  - PERSONAL_TOKEN: your github account token, not repository token. example: ghp_xxx
+  - **PERSONAL_TOKEN**: your GitHub account token/GitHub organization token, not repository token. example: 
+```
+ghp_xxx
+```
 - Add new variables with name
-  - YOUR_USERNAME_OR_ORGANIZATION_NAME: your username or organization name. example: paulpham157
-- # Cronjob default is weekly on Sunday at 00:00, change suitable for you in .github/workflows/sync.yml
+  - **YOUR_USERNAME_OR_ORGANIZATION_NAME**: your username or organization name. example: 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)